### PR TITLE
fix: Update peerDependencies to accept React v16.x.x

### PR DESCRIPTION
**Pull request by Xapphire13**
This package also works with React v16

### CODE REVIEW

**Comment by Xapphire13** [comment]
ping @theplatapi =]

**Comment by Xapphire13** [comment]
Any progress on getting this merged? Thanks

File: package.json [hunk 1]
```diff
@@ -21,8 +21,8 @@
   "license": "MIT",
   "homepage": "https://github.com/theplatapi/react-disqus-thread",
   "peerDependencies": {
-    "react": "^15.0.1",
-    "react-dom": "^15.0.1"
+    "react": ">=15.0.1 < 17.0.0",
```
Comment (theplatapi) [inline package.json:24]: Why not just list React 16?

Comment (Xapphire13) [reply package.json:24]: Didn't want to exclude v15. Happy to make any change you want though

Comment (Xapphire13) [reply package.json:24]: Are you suggesting to change the version to `^16.0.0`?

**Pull request merged**